ui/desktop/app/templates/application.hbs

```diff
@@ -1,17 +1,45 @@
 {{page-title (app-name)}}
 
-<Rose::Layout::Global as |layout|>
+<Rose::Layout::Global
+  class='{{if this.isWindowsOS ' layout-scroll'}}'
+  as |layout|
+>
 
   <layout.header>
-    <Rose::Header class='{{if (is-loading) ' loading'}}' as |header|>
+    <Rose::Header
+      class='{{if (is-loading) ' loading'}}
+        {{if this.isMacOS ' header-cushion'}}'
+      as |header|
+    >
       <header.brand @logo='logo-app' @text={{app-name}} />
-
       {{#if this.session.isAuthenticated}}
         <header.nav>
           {{outlet 'header-nav'}}
         </header.nav>
+      {{/if}}
 
-        <header.utilities as |utilities|>
+      <header.utilities as |utilities|>
+        {{#if this.isWindowsOS}}
+          <span class='utilities-actions'>
+            <Rose::Button
```

Comment:
Maybe instead of styling buttons within a container, we give these windowing buttons their own class name and style on that?  `button-desktop-windowing`?

Reply:
Added `button-window-minimize`, `button-window-fullscreen`, and `button-window-close` classes to Windows window actions to simplify and removed wrapper span.